app/ante/evm/fee_checker.go

```diff
@@ -36,32 +36,35 @@ func NewDynamicFeeChecker(k DynamicFeeEVMKeeper) authante.TxFeeChecker {
 			return checkTxFeeWithValidatorMinGasPrices(ctx, feeTx)
 		}
 
-		return feeChecker(ctx, k, feeTx)
+		return feeChecker(ctx, fmk, feeTx)
 	}
 }
 
 // feeChecker returns the effective fee and priority for a given transaction.
 func feeChecker(
 	ctx sdk.Context,
-	k DynamicFeeEVMKeeper,
+	k FeeMarketKeeper,
 	feeTx sdk.FeeTx,
 ) (sdk.Coins, int64, error) {
 	denom := config.GetEVMCoinDenom()
 	ethConfig := config.GetChainConfig()
-	baseFee := k.GetBaseFee(ctx, ethConfig)
-	if baseFee == nil {
+	baseFee := k.GetBaseFee(ctx)
+	if !types.IsLondon(ethConfig, ctx.BlockHeight()) || baseFee.IsNil() {
 		// london hardfork is not enabled: fallback to min-gas-prices logic
```

Comment:
```suggestion
```

Reply:
Here is not refected because "Github", but the code was updated to just ask about the London fork, so I think that we should keep the comment